composer.json

```diff
@@ -1,38 +1,47 @@
 {
     "name": "itk-dev/yesplan-asana",
     "description": "Application for ",
-    "type": "project",
     "license": "proprietary",
-    "minimum-stability": "stable",
-    "prefer-stable": true,
+    "type": "project",
     "require": {
         "php": ">=8.1",
         "ext-ctype": "*",
         "ext-iconv": "*",
-        "gedmo/doctrine-extensions": "^3.11",
         "doctrine/doctrine-bundle": "^2.8",
         "doctrine/doctrine-migrations-bundle": "^3.2",
         "doctrine/orm": "^2.14",
+        "gedmo/doctrine-extensions": "^3.11",
         "stof/doctrine-extensions-bundle": "^1.7",
-        "symfony/console": "6.2.*",
-        "symfony/dotenv": "6.2.*",
+        "symfony/console": "~6.2.0",
+        "symfony/dotenv": "~6.2.0",
         "symfony/flex": "^2",
-        "symfony/framework-bundle": "6.2.*",
-        "symfony/http-client": "6.2.*",
-        "symfony/mailer": "6.2.*",
+        "symfony/framework-bundle": "~6.2.0",
+        "symfony/http-client": "~6.2.0",
+        "symfony/mailer": "~6.2.0",
         "symfony/monolog-bundle": "^3.8",
-        "symfony/options-resolver": "6.2.*",
-        "symfony/runtime": "6.2.*",
-        "symfony/yaml": "6.2.*"
+        "symfony/options-resolver": "~6.2.0",
+        "symfony/runtime": "~6.2.0",
+        "symfony/yaml": "~6.2.0"
```

Comment:
I suppose this is the normalizer converting `6.2.*` to `~6.2.0`.

`6.2` is [EOL](https://symfony.com/releases) - should we upgrade to `^6.4`?

Reply:
We should. I'll do that in another pull request.